syncopy/tests/external/test_fooof.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
```

Comment:
As the fooof devs have their own tests, maybe we could really remove this? Although I like the `/external`, we might come back to this when interfacing MNE..

Reply:
yes, remove in 4d7687d